### PR TITLE
feature(#42): Add calendar

### PR DESCRIPTION
**Pull request by McFly1208**
Co-Authored-By: Enrique Mendoza <henrikhwolf@gmail.com>
Co-Authored-By: Karen Covarrubias <87504878+soykarencm@users.noreply.github.com>

# Description

## What?

- Calendar Component and add flights

## Why?

- This is necessary to choose the date on the flight to be booked.  See #42  for more information

## Testing

Follow the next steps to test the app:

1. Install modules

```
npm install
```
---

2. Now

Test app:

```
npm test
```
Run the app:

```
npm start
```

## Screenshots

| Changes made                                                                                                    |
| --------------------------------------------------------------------------------------------------------------- |
|imagen|

### CODE REVIEW

File: src/components/BookingCalendar.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            color={Styles.calendar.renderArrow.icon.color}
+          />
+        )}
+        onDayPress={(day) => {
+          this.props.formikProps.setFieldValue('inputs.date.value', day.dateString);
+        }}
+      />
+    );
+  }
+}
```
Comment (xManuel007) [inline src/components/BookingCalendar.tsx:40]: Add a blank line at the end

File: src/interfaces/Booking.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      error: boolean;
+    };
+    steps: {
+      value: number;
+      error: boolean;
+    };
+  };
+}
+
+export type IBookingFormikProps = FormikProps<IBookingState>;
```
Comment (xManuel007) [inline src/interfaces/Booking.ts:34]: Same here

File: src/interfaces/BookingCalendar.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+// Interfaces
+import { IBookingFormikProps } from './Booking';
+export interface IBookingCalendarInputsProps {
+  formikProps: IBookingFormikProps;
+}
```
Comment (xManuel007) [inline src/interfaces/BookingCalendar.ts:5]: and here

**Review by GaddPat0609 — APPROVED** [review]

File: src/screens/Booking.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+          {(formikProps) => (
+            <View>
+              <BookingCalendar formikProps={formikProps} />
+            </View>
+          )}
+        </Formik>
+      </ScrollView>
+    );
+  }
+}
```
Comment (xManuel007) [inline src/screens/Booking.tsx:59]: also here

**Review by EliasCode79 — APPROVED** [review]

File: src/styles/Booking.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+const screen = StyleSheet.create({
+  main: {
+    marginTop: Constants.statusBarHeight,
+  },
+});
+
+export default {
+  calendar,
+  screen,
+};
```
Comment (xManuel007) [inline src/styles/Booking.ts:49]: You must need to add a blank line at the end always

**Review by xManuel007 — CHANGES_REQUESTED** [review]
When you finish my request just send me a slack to remember me to approve

**Review by Zuack55 — APPROVED** [review]
nice job

**Review by xManuel007 — APPROVED** [review]
lgtm

**Pull request merged**